content/blog/2025/binder-singlenode/index.md

```diff
@@ -0,0 +1,72 @@
+
+While we're thinking through long-term ways to support Binder, 2i2c also wanted to provide short-term support for the project, and experiment with cheaper models for deploying Binder's reproducible environments. Making it simpler / easier to add capacity to the mybinder.org federation would both improve some of Binder's short-term capacity shortage, and may also provide an easier pathway for others to support the project in the future.
+
+So we're launching an experiment to deploy a Binder federation member via the simplest and cheapest cloud solution we could identify. The Kubernetes (and in particular, [k3s](https://k3s.io/)) ecosystem has matured, and there are some interesting cloud offerings that make a _single-node kubernetes cluster_ logistically accessible for workflows like mybinder.org. We're going to deploy a single-node Kubernetes cluster running mybinder.org sessions, hosted on [Hetzner](https://www.hetzner.com/cloud/). Here's a brief explainer for what we're trying, and why.
+
+## What if we could run a simple BinderHub federation member for cheap?
+
+Over the last year, two things have changed:
+
+**k3s lets us deploy and manage a very simple Kubernetes distribution**. In the last several months, we've been experimenting and developing experience in single-node Kubernetes workflows via [k3s](https://k3s.io/) (thanks to [Carl Boettiger](https://carlboettiger.info/) for collaborating on this with us!). [k3s](https://k3s.io/) is a lightweight Kubernetes distribution that is much easier to deploy and manage. It's designed for things like edge computing and low-resource environments, and it can be deployed with a single script! We think that we've got some expertise to deploy and manage this efficiently, and want to try this out as a simpler way to deploy a BinderHub instance..
+
+**Cheaper single-VM cloud offerings have become available**. As the cloud infrastructure space continues to grow, more service providers are competing on simplicity and price, which means there are cheaper options available. One-such provider is [Hetzner](https://hetzner.com/cloud), which provides a much cheaper single-VM option than other cloud providers. We'll get into this in the next section.  
```

Comment:
This paragraph should instead focus on 'single VM cloud offerings have become bigger', rather than available - I think hetzner's had some variant of single VM offering since before AWS started. But now you can get one that's 192GB RAM than just 32G of RAM - the ceiling has gone up. The pricing is relatively comparable - the big cloud providers have always been some multiple more expensive. 

Alternatively, we can focus this purely on k3s and what that enables.